webapp/channels/src/components/channel_settings_modal/channel_settings_configuration_tab.scss

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-@import "utils/_animations";
+@use "utils/animations";
```

Comment:
This change migrates from the deprecated @import to the modern @use syntax in Sass/SCSS.